SECURITY.md

```diff
@@ -0,0 +1,95 @@
+# Security Policy
+
+## Introduction
+
+Codenotary, September 7th 2022
+
+*This security policy is based on [this template](https://www.cisa.gov/vulnerability-disclosure-policy-template).*
+
+Codenotary is committed to ensuring the security of the American and global public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey our preferences in how to submit discovered vulnerabilities to us.
+
+This policy describes **what systems and types of research** are covered under this policy, **how to send** us vulnerability reports, and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.
+
+We encourage you to contact us to report potential vulnerabilities in our systems.
+
+## Authorization
+
+If you make a good faith effort to comply with this policy during your security research, we will consider your research to be authorized we will work with you to understand and resolve the issue quickly, and Codenotary will not recommend or pursue legal action related to your research. Should legal action be initiated by a third party against you for activities that were conducted in accordance with this policy, we will make this authorization known.
+
+## Guidelines
+
+Under this policy, “research” means activities in which you:
+
+- Notify us as soon as possible after you discover a real or potential security issue.
+- Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction or manipulation of data.
+- Only use exploits to the extent necessary to confirm a vulnerability’s presence. Do not use an exploit to compromise or exfiltrate data, establish persistent command line access, or use the exploit to pivot to other systems.
+- Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.
+- Do not submit a high volume of low-quality reports.
+
+Once you’ve established that a vulnerability exists or encounter any sensitive data (including personally identifiable information, financial information, or proprietary information or trade secrets of any party), **you must stop your test, notify us immediately, and not disclose this data to anyone else.**
+
+
+## Test methods
+
+The following test methods are not authorized:
+
+- Network denial of service (DoS or DDoS) tests or other tests that impair access to or damage a system or data
+- Physical testing (e.g. office access, open doors, tailgating), social engineering (e.g. phishing, vishing), or any other non-technical vulnerability testing
+
+## Scope
+
+This policy applies to the following systems and services:
+- immudb database core application, 

```

Comment:
Please add one extra empty line before and after bullet lists, otherwise it renders incorrectly